doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -0,0 +1,874 @@
+=================================
+Term rewriting and simplification
+=================================
+
```

Comment:
"Basic proof writing", why not indeed? The name "Proof writing" was chosen to accommodate the "Proof handling" section, but maybe it could have been "Basic tactics and proof management" instead? Anyway, we can debate / implement this change separately from this PR.